allennlp/service/predictors/constituency_parser.py

```diff
@@ -10,6 +11,52 @@
 from allennlp.data.tokenizers.word_splitter import SpacyWordSplitter
 
 
+# Make the links to POS tag nodes render as "pos",
+# to distinguish them from constituency tags. The
+# actual tag is still visible within the node.
+LINK_TO_LABEL = {x: "pos" for x in TAG_MAP.keys()}
```

Comment:
oh, I think I see what pylint wants. why do you use `.keys()` here? iterating over a dict already iterates over the keys

Reply:
I know, which is heinous in my opinion, and _always_ throws me when i'm reading code

Reply:
we say we roughly follow Google style guide, which doesn't like `.keys()`

https://google.github.io/styleguide/pyguide.html?showone=Default_Iterators_and_Operators#Default_Iterators_and_Operators

Reply:
fairsies